src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -7,6 +7,14 @@
  */
 public final class Constants {
 
+    public static final int USERNAME_MIN_LENGTH = 4;
+
+    public static final int USERNAME_MAX_LENGTH = 50;
+
+    public static final int PASSWORD_MIN_LENGTH = 4;
```

Comment:
Why is it 4 here? In one place below, (ManagedUserVm) it was 8 previously. In my opinion we should enforce 8 chars as a minimum length.

Reply:
```suggestion
    public static final int PASSWORD_MIN_LENGTH = 8;
```